plugins/fluent-bit/src/plugin.cpp

```diff
@@ -39,6 +81,130 @@ namespace tenzir::plugins::fluentbit {
 //     // No more JSON, but raw MsgPack delivery.
 //     flb_lib_push(ctx, in_ffd, msgpack_buf, msgpack_buf_len);"
 
+/// Utilities wrapping the MsgPack C API.
+namespace msgpack {
+
+auto to_str(const msgpack_object& object) {
+  return std::string_view{object.via.str.ptr, object.via.str.size};
+}
+
+auto to_array(const msgpack_object& object) {
+  return std::span<msgpack_object>{object.via.array.ptr,
+                                   size_t{object.via.array.size}};
+}
+
+auto to_map(const msgpack_object& object) {
+  return std::span<msgpack_object_kv>{object.via.map.ptr,
+                                      size_t{object.via.map.size}};
+}
+
+auto to_bin(const msgpack_object& object) {
+  return std::span<const std::byte>{
+    reinterpret_cast<const std::byte*>(object.via.bin.ptr),
+    size_t{object.via.bin.size}};
+}
+
+/// A MsgPack object.
+auto visit(auto f, const msgpack_object& object) {
```

Comment:
`default` case missing here - intended?

Reply:
Yes. In this case the compiler will warn me if I forgot an enum.